app/src/main/java/com/android/sample/ui/workout/WorkoutScreen.kt

```diff
@@ -262,6 +269,14 @@ fun WorkoutScreenBody(
     } else if (!summaryScreenIsDisplayed) {
       summaryScreenIsDisplayed = true
     } else {
+      // delete the workout
+      val stats =
+          statisticsViewModel.computeWorkoutStatistics(
+              workout = workout,
+              exerciseList = exerciseStateList ?: emptyList(),
+              userAccountViewModel = userAccountViewModel)
+      statisticsViewModel.addWorkoutStatistics(stats)
+      workoutViewModel.deleteWorkoutById(workoutID)
```

Comment:
Wait why do you delete it ? The user maybe want to redo his workout another day, or yeah maybe im missing something.(The structure of our data may cause some problem as we store a workout with a date, maybe we have to modify something to get some "workout skeleton", that have no date to be able to the user to create the same workout easily) 
We can imagine that in the import feature, we can have an option like "latest workout" to be able to do a workout recently done

 We need to discuss it.

Reply:
As discussed I will implement some methods to tranfer workouts in another collection like "done" instead of completely deleting them. So that we can also import a done workout.